lib/paper_trail.rb

```diff
@@ -46,124 +47,158 @@ module PaperTrail
   class << self
     # @api private
```

Comment:
Does this comment actually do anything or it's merely a note to help developers remember which methods are public / private?

Reply:
It specifies whether something is part of our public API, ie. whether we would consider changing it to be a breaking change. Notably, not all `public` methods are public API, if that makes sense.